TelligentInstall/solr.ps1

```diff
@@ -92,4 +136,33 @@ function Remove-SolrCore {
     }
 }
 
+function Remove-SolrCore {
+    <#
+    .SYNOPSIS
+        Removes a Solr Core
+    .PARAMETER Name
+        The name of the Solr Core to remove
+    .PARAMETER CoreBaseDir
+        The path to the root of the Solr instance hosting the core
+    .PARAMETER CoreAdmin
+        The url to the solr instance's Core Admin API.
+    #>
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true, Position=0)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Name,
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [Uri]$CoreAdmin
+    )
+
+    $url = "${CoreAdmin}?action=UNLOAD&core=${Name}-content&deleteindex=true"
+    Invoke-WebRequest $url -UseBasicParsing -Method Post | Out-Null
+
+    $url = "${CoreAdmin}?action=UNLOAD&core=${Name}-conversations&deleteindex=true"
+    Invoke-WebRequest $url -UseBasicParsing -Method Post | Out-Null
```

Comment:
Ok, I was just truing to keep knowledge of Solr in this Solr file so we can handle that with version logic. That said, we don't currently know the version in this method so it would require us passing that in (either the number or the package like elsewhere).